src/mixins/canvas_grouping.mixin.js

```diff
@@ -74,6 +74,8 @@
       var currentActives = this.getActiveObjects(), group = this._createGroup(target);
       this._hoveredTarget = group;
       // ISSUE 4115: should we consider subTargets here?
```

Comment:
@asturur not really sure what to do here

Reply:
we can skip for now. There are already enough moving pieces.